assets/template-collection.css

```diff
@@ -447,7 +474,7 @@ a.active-facets__button:focus {
 
```

Comment:
Not related to this PR, but facet filtering. I noticed that the button has an uneven focus at the top and bottom:

https://screenshot.click/05-42-rfdgj-17eqk.png

Reply:
I'll add it to the issue as a minor bug.